tests/test_product_mixin.py

```diff
@@ -0,0 +1,87 @@
+import pytest
+from telluric import GeoRaster2
+
+from common_for_tests import (
+    micro_raster_8b, sensor_bands_info,
+    macro_raster, macro_raster_with_no_data,
+    micro_raster_with_no_data, micro_raster_16b
+)
+
+from telluric.products import ProductsFactory
+
+
+def test_micro_raster_gets_available_products():
+    products = micro_raster_8b().get_products(sensor_bands_info())
```

Comment:
don't use the word, micro the officail camera name is multispectral

for macro is Hyperspectral